### ISSUE TITLE
Ingress Controller Not Creating VIP for LoadBalancer Services

### ISSUE THREAD

**Issue by travissutton** [citrix/citrix-k8s-ingress-controller#91]
I've setup a single-tier deployment using a HA NS VPX configuration and kubernetes v1.14.3. When I go to create a service and assign the `loadbalancer` spec type, the ingress-controller registers the event but determines that it is not responsible for creating the VIP? Here is the relevant logs: https://pastebin.com/SpSDBUPQ. 

I am using the IPAM setup. It is communicating with the NS VPX configuration correctly as the watcher did create the node network routes.

I just haven't seen any info about setting up past this point to know if there is some prioritization or something that needs to be done. Thanks in advance.

**Comment by janraj** [comment]
@travissutton, We need more information to identify the issue. Can you please provide the environment variables [Optional arguments except Username/Password/IP's] provided in CIC yamls   Would it possible for you to join our slack channel ? Please refer Questions section on the main page [https://github.com/citrix/citrix-k8s-ingress-controller]

**Comment by travissutton** [comment]
@janraj Sure. I'll join the slack channel. I think I just found the issue with the configuration while pulling the environment variables. I noticed that my `--ipam=citrix-ipam-controller` wasn't correct and so updated that in the helm chart I modified. This formatted it correctly and it seems that it is working now. So it appears that it was just not getting an IP address instead - and since I was trying to get one from the IPAM, I was not specifying on in the annotation.

Either way, I'm looking forward to the integration. Sorry for the errant issue.
